content/news/2024-07-01-GCC2024/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Galaxy Community Conference 2024 - a brief summary!
+date: '2024-07-01'
+tease: A brief summary for GCC 2024
+tags: [CoFest, training, GCC, Elixir, EOSC, 2024, Brno]
+supporters:
+- JXTX Foundation
+- Worfklow4Metabelomics
+- GalaxyWorks
+- GigaScience
+- KWS
+- Limagrain
+- deNBI
+subsites: [all-eu, global, us]
+main_subsite: eu
+---
+
+
+The [2024 Galaxy Community Conference (GCC2024)](https://galaxyproject.org/events/gcc2024/) was held in Brno, Czech Republic, 24-29 June, 2024. 151 participants from 28 different countries, 54 talks, 55 posters/demos, 14 Birds of a Feather and 80 CoFest participants.
+
+![GCC2024 Logo](gcc2024logo.png)
+![GCC2024 Group Picture](gcc2024groupphoto.png)
+
+
+## Keynotes
+
+The conference featured notable keynote speakers, including Jana Klánová from Masaryk University, who discussed the integration of exposomics into biomedical sciences. She highlighted the importance of studying environmental exposures and their effects on health, introduced RECETOX, and discussed European initiatives like PARC and EIRENE that support exposome research. Ute Gunsenheimer, Secretary General of the EOSC Association, spoke on building the EOSC Federation and the potential role of Galaxy. Dr. Daria Onichtchouk from the University of Freiburg shared her extensive use of the Galaxy platform in her research on the genetic control of pluripotency, focusing on the maternal to zygotic transition in zebrafish and its underlying mechanisms, including chromatin accessibility and nucleosome affinity.
+
+## Talks Sessions
```

Comment:
I would not sub divide by 'Single Track', 'Analysis Track' and 'Enablement Track'. Just list all the Freiburg talks (no matter which session they were in).

Reply:
I can do that, once they are written I will just merge them into one